doc/user_guide/user_guide.md

```diff
@@ -358,7 +358,7 @@ You can download the model using python like this:
         # download the model and tokenizer from Hugging Face
         model = model_factory.from_pretrained(model_name)
         # save the downloaded model using the save_pretrained function
-        model_save_path = <save_path> / "pretrained" / <model_name>
+        model_save_path = <save_path> / "pretrained" / <model_name> #todo do we use the path function here? or should this be using none of our functions?
```

Comment:
To be honest, we probably need to get rid of this, the user shouldn't need to handle any of the weird paths

Reply:
changed in docu. path does not need to be specified by us here